Makefile

```diff
@@ -1,23 +1,8 @@
-PROGRAM_EMAIL = email-cli
-PROGRAM_UPS = ups-cli
-PROGRAM_UPSXX = ups-cli++
-PROGRAM_MON = monitor-cli
-PROGRAM_ZYRE_TRIVIAL = zyre-trivial
-PROGRAM_MALAMUTEZ = malamute-z
-
-PROGRAMS_C = $(PROGRAM_EMAIL) $(PROGRAM_UPS) $(PROGRAM_ZYRE_TRIVIAL)
-# $(PROGRAM_MALAMUTEZ)
-PROGRAMS_CXX = $(PROGRAM_MON) $(PROGRAM_UPSXX)
-PROGRAMS = $(PROGRAMS_C) $(PROGRAMS_CXX)
-
-#SOURCES_C = $(addsuffix .c,$(PROGRAMS_C))
-#SOURCES_CXX = $(addsuffix .cc,$(PROGRAMS_CXX))
-
-RM = rm -f
```

Comment:
Apparently lack or disablement of this built-in definition is what broke for those who could not use "make clean", weird...